kit/Delta.hpp

```diff
@@ -242,7 +242,7 @@ class DeltaGenerator {
 
         void initRow(const uint32_t *from, unsigned int width)
         {
-            uint32_t scratch[width];
+            auto scratch = static_cast<uint32_t*>(alloca(sizeof(uint32_t) * width));
```

Comment:
Or I guess we could turn the warning off?

Reply:
That's also an option, though searching github, it seems some other big projects just adapt similarly, https://github.com/dotnet/diagnostics/commit/d592e0da173e520ef42d4f9cdd9d7b138a9896fd is a sample .NET commit e.g.